terraform/strings.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package terraform
```

Comment:
I'd create this as a file and copy it to the resources using a provisioner as I mentioned

Reply:
See comment above.